src/content/docs/cypher/import-export-db.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Migrating databases
+---
+
+You can migrate databases between different versions without manually writing DDL and COPY statements.
+Note that because the storage layer of Kùzu can change between versions, the import/export is not
+guaranteed to be compatible between versions of Kùzu that are very far apart.
+
+## Export database
+
+The `EXPORT DATABASE` command allows you to export the contents of the database to a specific directory.
+The query below exports the database to an absolute directory named `/path/to/export`, utilizing the same configuration parameters as `COPY FROM` statements.
+```cypher
+EXPORT DATABASE '/path/to/export' (HEADER=true);
```

Comment:
fromat is another option that we can provide during export (format can be parquet or csv)

Reply:
Makes sense, I included that too.